mdoc/src/main/scala/mdoc/PostModifier.scala

```diff
@@ -1,19 +1,25 @@
 package mdoc
 
 import java.util.ServiceLoader
-import mdoc.internal.cli.Settings
+
+import mdoc.internal.cli.{Exit, Settings}
 import metaconfig.ConfDecoder
 import metaconfig.ConfEncoder
 import metaconfig.ConfError
 import metaconfig.generic.Surface
+
 import scala.meta.inputs.Input
 import scala.meta.io.AbsolutePath
 import scala.collection.JavaConverters._
 import scala.meta.io.RelativePath
 
 trait PostModifier {
   val name: String
+  def onStart(settings: MainSettings): Unit = ()
```

Comment:
Can we add docstrings to all the new methods? It would be good to have some ascii diagram or flow chart or something like that to visualize when what methods are called when (doesn't have to be fancy ascii art)

Reply:
Added docstring. Did not add the "ascii diagram or flow chart" because we only have a *start* and *stop* now. Added instead a small description to the trait.